src/js/actions/index.js

```diff
@@ -426,16 +426,21 @@ export function fetchPurchaseOrderStatuses() {
   };
 }
 
-export function fetchPaymentTerms() {
-  return (dispatch) => {
-    purchaseOrderApi.getPaymentTerms().then((res) => {
-      dispatch({
-        type: FETCH_PAYMENT_TERMS,
-        payload: res.data.data,
-      });
-    });
+export const fetchPaymentTerms = ({ translate }) => async (dispatch) => {
+  const blankPaymentTermOption = {
+    id: 'null',
+    value: 'null',
+    label: translate?.(
+      'react.purchaseOrder.column.blankPaymentTermOption.label',
+      'Blank Payment Term',
+    ),
   };
-}
+  const response = await purchaseOrderApi.getPaymentTerms();
+  return dispatch({
+    type: FETCH_PAYMENT_TERMS,
+    payload: [blankPaymentTermOption, ...response?.data?.data],
```

Comment:
If one part from the chaining: ` ...response?.data?.data` will be null your payload will look like: `[blankPaymentTermOption, '']`. Is it intended?

Reply:
if I return `blankPaymentTermOption` from the api like @kchelstowski suggested, it shouldn't be an issue, but I guess as it is currently it might have been an issue so thanks.